client/reader/stream/index.jsx

```diff
@@ -365,9 +449,13 @@ export default class ReaderStream extends React.Component {
 	}
 
 	renderPost = ( postKey, index ) => {
-		let postState,
-			PostClass;
-		const isSelected = index === this.state.selectedIndex;
+		const selectedPostKey = this.props.store.getSelectedPost();
+		const isSelected = selectedPostKey &&
+			selectedPostKey.postId === postKey.postId &&
+			(
```

Comment:
no, should be two different post key instances. that said, there's no guarantee that there's only one post key instance for a given id pair. We can't just use `===` to do the compare.

a post key has _either_ a blogId or a feedId, but never both. In retrospect, we probably should have used a different property for each site for the post id. But, future PR as that will touch a ton of stuff.